pyterrier_rag/backend/_base.py

```diff
@@ -204,6 +206,12 @@ def __init__(self,
         self.batch_size = batch_size
         self.max_new_tokens = max_new_tokens
         self.num_responses = num_responses
+    
+    def transform_inputs(self) -> List[List[str]]:
+        return [[self.input_field]]
+    
+    def transform_outputs(self, inp_cols):
+        return inp_cols + [self.output_field]
```

Comment:
This needs to perform validation to ensure that the inp_columns are valid. Also, it should probably check whether self.output_field is already in inp_columns. (When it is, it overwrites it.)